package.json

```diff
@@ -1,23 +1,25 @@
 {
-  "name": "how",
-  "version": "0.1.0",
-  "description": "Ask a cli tool about other cli tools.",
-  "module": "./src/index.ts",
-  "type": "module",
-  "scripts": {
-    "how": "bun run ./src/index.ts --",
-    "compile": "bun build ./src/index.ts --compile --minify --sourcemap --outfile how && chmod +x ./how",
-    "replace-bin": "sudo mv ./how /usr/local/bin/how"
-  },
-  "dependencies": {
-    "@inquirer/prompts": "^5.1.2",
-    "chalk": "^5.3.0",
-    "clipboardy": "^4.0.0"
-  },
-  "devDependencies": {
-    "@types/bun": "latest"
-  },
-  "peerDependencies": {
-    "typescript": "^5.0.0"
-  }
+	"name": "how",
+	"version": "0.1.0",
+	"description": "Ask a cli tool about other cli tools.",
+	"module": "./src/index.ts",
+	"type": "module",
+	"scripts": {
+		"how": "bun run ./src/index.ts --",
+		"compile": "bun build ./src/index.ts --compile --minify --sourcemap --outfile how",
+		"check": "biome check ./src --fix"
+	},
+	"dependencies": {
+		"@inquirer/prompts": "^5.1.2",
+		"chalk": "^5.3.0",
+		"clipboardy": "^4.0.0",
+		"env-paths": "^3.0.0"
+	},
+	"devDependencies": {
+		"@types/bun": "latest",
+		"biome": "^0.3.3"
```

Comment:
i use zed editor so i opted in by default to prettier, i will check out biome and see if i like it. seems nice

Reply:
There's a biome plugin for zed :)

Reply:
ok i tried biome. 
* didn't see much performance gain compared do prettier formatting (so this is not a killer feature for me)
* i like that compared to eslint it is only one package, but
* i don't like may of the rules and i don't want to go and disable them manually
 
for such a small projects i don't see a need in linters, they are more of a distraction